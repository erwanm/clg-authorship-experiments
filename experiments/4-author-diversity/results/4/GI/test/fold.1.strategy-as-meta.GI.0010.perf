0.318630	0.559000	0.570000
