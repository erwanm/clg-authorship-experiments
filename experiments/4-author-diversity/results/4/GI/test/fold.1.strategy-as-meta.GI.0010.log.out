apply-config.sh: applying config '/tmp/expe4/4/process/selected-meta-configs/fold.1.strategy-as-meta.GI.0010' to cases '/home/moreaue/dump/expe4//results/4/GI/test/only-cases.tmp.zK1BpWuV'; results written to '/home/moreaue/dump/expe4//results/4/GI/test/fold.1.strategy-as-meta.GI.0010'
apply-config.sh: done.
